### PR TITLE
add Desktop button

### PR DESCRIPTION
**Pull request by dashapenkova**

### CODE REVIEW

**Review by romanskaskiv — COMMENTED** [review]
I watched. I am single

File: perfectTestWishList/src/main/java/pages/Header.java [hunk 1]
```diff
@@ -24,4 +25,7 @@ public WishListPage goToWishList(){
         click(buttonWishList);
         return new WishListPage(driver);
     }
+
```
Comment (ON-OR) [inline perfectTestWishList/src/main/java/pages/Header.java]: Darynka,
Could you please delete those three extra lines.

File: perfectTestWishList/src/main/java/pages/Header.java [hunk 2]
```diff
@@ -24,4 +25,7 @@ public WishListPage goToWishList(){
...
+
```
Comment (dganushkeevych) [inline perfectTestWishList/src/main/java/pages/Header.java]: _Dasha, fix it please_

File: perfectTestWishList/.idea/misc.xml [hunk 3]
```diff
@@ -8,7 +8,7 @@
       </list>
     </option>
   </component>
-  <component name="ProjectRootManager" version="2" languageLevel="JDK_12" project-jdk-name="11" project-jdk-type="JavaSDK">
+  <component name="ProjectRootManager" version="2" languageLevel="JDK_12" project-jdk-name="11.0.4" project-jdk-type="JavaSDK">
```
Comment (karrtopelka) [inline perfectTestWishList/.idea/misc.xml:11]: Not bad, not bad.

**Review by karrtopelka — COMMENTED** [review]
Now you

File: perfectTestWishList/src/main/java/pages/Header.java (hunk 2 above)
Comment (karrtopelka) [reply perfectTestWishList/src/main/java/pages/Header.java]: NNo

File: perfectTestWishList/src/main/java/pages/Header.java [hunk 4]
```diff
@@ -24,4 +25,7 @@ public WishListPage goToWishList(){
...
+
```
Comment (ON-OR) [inline perfectTestWishList/src/main/java/pages/Header.java]: Darynka Penkova,
Could you please delete these lines cause they are useless

Comment (karrtopelka) [reply perfectTestWishList/src/main/java/pages/Header.java]: Orest yhodi

**Review by romanskaskiv — COMMENTED** [review]
Dasha, it was your biggest mistake. I wish it could never happened. Shame on you!

**Review by ON-OR — APPROVED** [review]

**Review by karrtopelka — APPROVED** [review]

**Pull request closed without merging**